Tell Travis to begin testing on Ruby 2.2.x

myabc opened the pull request:
Thanks to @marutosi for alerting us to the Ruby 2.2.0 release!

myabc committed:
Tell Travis to begin testing on Ruby 2.2.x

Thanks to @marutosi for alerting us to the Ruby 2.2.0 release!

Signed-off-by: Alex Coles <alex@alexbcoles.com>

TeatroIO commented:
I've prepared a stage. <a href='http://feature-ruby-2-2.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

myabc committed:
Tell Travis to run frontend tests only once

Signed-off-by: Alex Coles <alex@alexbcoles.com>

Commit:
Update mysql2 0.3.17

myabc committed:
Bundle update thin

Signed-off-by: Alex Coles <alex@alexbcoles.com>

myabc committed:
Gemfile: Depend on syck on ruby_22 too

Signed-off-by: Alex Coles <alex@alexbcoles.com>

myabc commented:
This PR relies on bundler/bundler#3328 being merged.

myabc committed:
Gemfile: Specify syck version, Bump to ~> 1.0.4

1.0.4 contains a fix for Ruby 2.2.0:
https://github.com/tenderlove/syck/commit/1f148c39cf9f538c0b2a54ef0289b65894973600

Signed-off-by: Alex Coles <alex@alexbcoles.com>

myabc committed:
Tell Travis to update Bundler (temporarily)

We need Bundler 1.7.11+ to support Ruby 2.2 platform:
https://github.com/bundler/bundler/pull/3309
https://github.com/bundler/bundler/pull/3328

Signed-off-by: Alex Coles <alex@alexbcoles.com>

myabc committed:
Switch to rails from git (3-2-stable branch)

Pending a release containing Ruby 2.2 patches:
https://github.com/rails/rails/pull/18306

Signed-off-by: Alex Coles <alex@alexbcoles.com>

Commit:
Revert "Tell Travis to update Bundler (temporarily)"

This reverts commit e0298f7196ac2d53e9692cecf5d457c314ec8066.

https://github.com/redmine/redmine/blob/177dd7bff13c3f41a0e4/.travis.yml#L9

Commit:
Merge branch 'dev' into feature/ruby-2.2

linki committed:
Merge branch 'dev' into feature/ruby-2.2

Conflicts:
	Gemfile.lock

linki commented:
took the liberty to merge in the latest `dev`. not sure if i did everything right, but you can always use your old hash if you like.

myabc committed:
Merge branch 'dev' into feature/ruby-2.2

Signed-off-by: Alex Coles <alex@alexbcoles.com>

Conflicts:
	Gemfile.lock

myabc committed:
Bundle update rails

Signed-off-by: Alex Coles <alex@alexbcoles.com>

myabc committed:
Merge branch 'feature/remove-test-unit' into feature/ruby-2.2

Signed-off-by: Alex Coles <alex@alexbcoles.com>

Conflicts:
	Gemfile.lock

myabc committed:
Add minitest as dependency

Necessary as minitest no longer comes bundled with stdlib, as of Ruby
2.2.

myabc committed:
Temp. add back test-unit dependency

myabc committed:
Make Travis execute jobs on Ruby 2.2 first

Signed-off-by: Alex Coles <alex@alexbcoles.com>

myabc committed:
Merge pull request #2775 from marutosi/finn/ruby-2.2/revert-bundler-only

Revert "Tell Travis to update Bundler (temporarily)"
